tests/queries/0_stateless/00804_test_custom_compression_codecs.sql

```diff
@@ -9,18 +10,20 @@ CREATE TABLE compression_codec(
     ddd Date CODEC(NONE),
     somenum Float64 CODEC(ZSTD(2)),
     somestr FixedString(3) CODEC(LZ4HC(7)),
-    othernum Int64 CODEC(Delta)
+    othernum Int64 CODEC(Delta),
+    qplstr String CODEC(DEFLATE_QPL),
```

Comment:
The column names are supposed to stay clear of the column codec. Let's do this:]

```sql
CREATE TABLE compression_codec(
    id UInt64 CODEC(LZ4),
    data String CODEC(ZSTD),
    ddd Date CODEC(NONE),
    ddd32 Date32 CODEC(DEFLATE_QPL),
    somenum Float64 CODEC(ZSTD(2)),
    somestr FixedString(3) CODEC(LZ4HC(7)),
    othernum Int64 CODEC(Delta)
) ENGINE = MergeTree() ORDER BY tuple();
```

Reply:
Done